.github/workflows/dispatch-build-bottle.yml

```diff
@@ -40,9 +40,17 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
+            const intelMacOSRegex = /^\d+(?:\.\d+)?(?:-arm64)?$/;
```

Comment:
Don't you want a negative look-ahead assertion for the `-arm64` since this is called `intelMacOSRegex`? (Don't know if our regex here supports that tho)

Reply:
Not entirely sure why I typed that bit in the first place.

Don't even need a negative lookahead because of the `$`.